Python API segmentation fault

### Commit
59f80849ff

### Steps to reproduce
1. Execute demo_program.py

### Observed behavior
Everything is excecuted as expected, but at the end segmentation fault is reported.

### Logfiles
```
Executing /home/gsansone/catkin_ws/src/pilz_industrial_motion/pilz_robot_programming/examples/demo_program.py
[INFO] [1587549598.452606]: Waiting for connection to action server sequence_move_group...
[INFO] [1587549598.696596]: Waiting for connection to service /prbt/get_speed_override...
[INFO] [1587549598.702237]: Connection to service /prbt/get_speed_override estabilshed
[INFO] [1587549598.705450]: Move: Ptp
[ERROR] [1587549598.729616803]: Virtual joint does not attach to a link on the robot (link 'world' is not known)
[ INFO] [1587549598.731583855]: Loading robot model 'prbt'...
[ INFO] [1587549598.731655205]: No root/virtual joint specified in SRDF. Assuming fixed joint
[ INFO] [1587549598.774270665]: Using link_prefix: 'prbt_'
[INFO] [1587549598.778157]: RobotCommander created.
[ INFO] [1587549599.985323657]: Ready to take commands for planning group manipulator.
[INFO] [1587549602.481543]: Move: Ptp
[INFO] [1587549604.523664]: Move: Ptp
[INFO] [1587549605.809656]: Move: Ptp
[INFO] [1587549607.062826]: Move: Lin
[INFO] [1587549608.918050]: Move: Lin
[INFO] [1587549611.582124]: Move: Lin
[INFO] [1587549614.251475]: Move: Circ
[INFO] [1587549616.921181]: Move: Ptp
[INFO] [1587549622.365104]: Move: Sequence
[INFO] [1587549631.075637]: Move: Ptp
[INFO] [1587549632.337130]: Move: Sequence
[INFO] [1587549637.470976]: Move: Ptp
[INFO] [1587549638.520444]: Move: Ptp
[INFO] [1587549639.361015]: Move: Ptp
[ERROR] [1587549639.413600]: Failure during execution of: Ptp vel_scale: 1.0 acc_scale: 1.0 reference: prbt_base joint goal: [0, 10.0, 0, 0, 0, 0]
[INFO] [1587549639.417148]: Ptp command did fail as expected.
Segmentation fault (core dumped)
```

--- a/pilz_robot_programming/src/pilz_robot_programming/robot.py
+++ b/pilz_robot_programming/src/pilz_robot_programming/robot.py
@@ -446,6 +446,8 @@ class Robot(object):
             first_iteration_flag = False
 
     def _on_shutdown(self):
+        if self.__robot_commander is not None:
+            del self.__robot_commander
         with self._move_ctrl_sm:  # wait, if _execute is just starting a send_goal()
             actionclient_state = self._sequence_client.get_state()
         # stop movement

